app/src/main/java/com/blackboxteam/app/blackbox/LoginScreenActivity.java

```diff
@@ -0,0 +1,82 @@
+package com.blackboxteam.app.blackbox;
+
+import android.content.Intent;
+import android.support.design.widget.Snackbar;
+import android.support.design.widget.TextInputLayout;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.blackboxteam.app.blackbox.util.Validation;
+
+import static com.blackboxteam.app.blackbox.util.Validation.validateEmail;
+import static com.blackboxteam.app.blackbox.util.Validation.validateMinLength;
+
+public class LoginScreenActivity extends AppCompatActivity implements View.OnClickListener {
+
+    Button loginBtn;
+    EditText etEmail, etPassword;
+    TextInputLayout tiEmail, tiPassword;
+    Validation validate;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login_screen);
+
+        loginBtn = (Button) findViewById(R.id.btn_login);
+        etEmail = (EditText) findViewById(R.id.et_email);
+        etPassword = (EditText) findViewById(R.id.et_password);
+        tiEmail = (TextInputLayout) findViewById(R.id.ti_email);
+        tiPassword = (TextInputLayout) findViewById(R.id.ti_password);
+        loginBtn.setOnClickListener(this);
+    }
+
+    private void login() {
+        setError();
+        String email = etEmail.getText().toString();
+        String password = etPassword.getText().toString();
+
+        int err = 0;
+
+        if (!validateEmail(email)) {
+            err++;
+            tiEmail.setError("Email should be valid");
+        }
+
+        if (!validateMinLength(password, 6)) {
+            err++;
+            tiPassword.setError("Password must be greater than 5");
+        }
+
+        if (err == 0) {
+            Intent intent = new Intent(LoginScreenActivity.this, ActivateBlackboxScreenActivity.class);
+            startActivity(intent);
+        } else {
+            showSnackBarMessage("Enter Valid Details !");
+        }
+    }
+
+    private void setError() {
```

Comment:
The function is used to set the error message for textInputEmail and textInputPassword to null, more like clearing the errors. I'll make the function more descriptive